docs/application/web/guides/connectivity/download.md

```diff
@@ -138,7 +158,7 @@ To provide the user access to Internet resources, you must learn how to check th
    ```
 
 ## Related Information
-* Dependencies   
+- Dependencies
```

Comment:
why did you change the enumeration type?

Reply:
@pkosko 

This enum change is as per our style guide. 
Kindly refer to [Styleguide](https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#document-layout)

Also while parsing and converting our MD format to HTML, sometimes if we use any other symbol(+,*)  other than (-) conversion didn't go well sometimes.

Reply:
ok